defaults/test_default.yaml

```diff
@@ -35,3 +35,6 @@ cloud_credentials_path: '~/.ssh/support'
 backtrace_decoding: false
 
 logs_transport: "rsyslog"
+
+send_email: false
```

Comment:
@aleksbykov what Roy meant to set `send_email: true` and set false to all that don't support sending emails. Please fix. Thanks